cmd/crypto/header.go

```diff
@@ -111,13 +113,13 @@ func (ssec) IsRequested(h http.Header) bool {
 // at least one SSE-C copy header. Regular SSE-C headers
 // are ignored.
 func (ssecCopy) IsRequested(h http.Header) bool {
-	if _, ok := h[SSECopyAlgorithm]; ok {
+	if h.Get(SSECopyAlgorithm) != "" {
 		return true
 	}
-	if _, ok := h[SSECopyKey]; ok {
+	if h.Get(SSECopyKey) != "" {
 		return true
 	}
-	if _, ok := h[SSECopyKeyMD5]; ok {
+	if h.Get(SSECopyKeyMD5) != "" {
```

Comment:
Not correct. The header can be set but be empty.

Reply:
fixed